docs/development/getting_started.rst

```diff
@@ -145,7 +145,7 @@ development, staging, and prod (production).
    hardening applied but virtualized
 
 If you plan to alter the configuration of any of these machines, make sure to
-review the `Development Guide for Serverspec Tests </docs/spec_tests.md>`__.
+review the `Development Guide for Serverspec Tests </docs/development/spec_tests.rst>`__.
```

Comment:
Ack. My bad. I'll dive into building with Sphinx so I know what I'm doing. I was curious about that - I had only checked that links worked when browsing via GitHub.com. Thanks for the explanation.